nix/shell.nix

```diff
@@ -43,6 +43,7 @@ craneLib.devShell {
 
     # documentation utilities
     swagger-codegen3
+    vacuum-go
```

Comment:
This also needs to be added to the derivation dependency to make running the tests in CI work, because those don't use the devshell.

Reply:
Can you point me in the right direction pls?

Reply:
I removed the vacuum-go, since it was not used in any test.